website/docs/api/admin/feature-toggles-api-v2.md

```diff
@@ -599,3 +635,130 @@ http PATCH http://localhost:4242/api/admin/projects/default/features/demo/varian
 }
 ```
 
+
+## Manage project users and roles
+
+You can add and remove users to a project using the `/api/admin/projects/:projectId/users/:userId/roles/:roleId` endpoint. When adding or removing users, you must also provide the ID for the role to give them (when adding) or the ID of the role they currently have (when removing).
+
+There is no way to update a user's role directly at the moment. Instead, if you want to change a user's role, first remove the user from the project and then add them back with the desired role.
+
+### Add a user to a project
+
+``` http
+POST /api/admin/projects/:projectId/users/:userId/roles/:roleId
+```
+
+This will add a user to a project and give the user a specified role within that project.
+
+#### URL parameters
+
+| Parameter   | Type    | Description                                                           | Example value     |
+|-------------|---------|-----------------------------------------------------------------------|-------------------|
+| `userId`    | integer | The ID of the user you want to add to the project.                     | `1`               |
+| `projectId` | string  | The id of the project to add the user to.                              | `"MyCoolProject"` |
+| `roleId`    | integer | The id of the role you want to assign to the new user in the project. | `7`                  |
+
+
+#### Responses
+
+<details>
+<summary>Responses data</summary>
+
```

Comment:
What's your thoughts on using the `details` element for response data?

Reply:
Works for me :)